indexing-service/src/main/java/io/druid/indexing/common/task/AbstractTask.java

```diff
@@ -129,6 +129,19 @@ public String getClasspathPrefix()
   }
 
   @Override
+  public boolean canRestore()
+  {
+    return false;
+  }
+
+  @Override
+  public void stopGracefully()
+  {
+    // Should not be called when canRestore = false.
```

Comment:
not sure if we can do anything useful here even if canRestore() returned true. any implementation has to override both of them together.